config/script.go

```diff
@@ -63,7 +80,14 @@ func EvalScript(evalctx *eval.Context, script hcl.Script) (Script, error) {
 		evaluatedJob := ScriptJob{}
 
 		if job.Command != nil {
-			command, err := evalScriptJobCommand(evalctx, job.Command.Expr, "command")
+			expr := job.Command.Expr
+			v, err := evalctx.Eval(expr)
+			if err != nil {
+				errs.Append(errors.E(ErrScriptSchema, expr.Range(), err, "evaluating command"))
+				continue
+			}
+
+			command, err := unmarshalScriptJobCommand(v, expr)
```

Comment:
nitpick: the "unmarshaling" name sounds a bit strange for me.

Reply:
I also thought about this for a while, but I couldn't come up with anything better. Technically, I would say it's a type cast from cty.Value to a concrete type, but `castScriptJobCommand` also sounds strange. If you have any suggestions let me know.

Reply:
Marking this as resolved for now, can refactor if we come up with a better name.